homeassistant/components/climate/daikin.py

```diff
@@ -0,0 +1,260 @@
+"""
+Support for the Daikin HVAC.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.daikin/
+"""
+import re
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.climate import (
+    ClimateDevice,
+    SUPPORT_TARGET_TEMPERATURE,
```

Comment:
Please keep as many names per line that will fit the line.

Reply:
I mean ClimateDevice refers to something, while SUPPORT_*** constants can be considered a different league.

Can I at least group them @MartinHjelmare or we really want them chained one after another even if they are not related ?

Reply:
Yes please condense everything within the same group. It would be best if you sort the names 🔡, but that isn't strictly necessary.

Reply:
Ok, done :)